### PR TITLE
TASK-51872: can't remove start date of a task

### PR DESCRIPTION
**Pull request by hthabti**
issue: start date can't be removed when clicking on 'none' label from the datepicker
fix: add else-if statement to handle user's input on the datepicker when the value of entered label = 'none'

### CODE REVIEW

File: task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue [hunk 1]
```diff
@@ -482,6 +482,17 @@ export default {
         } else {
           this.taskStartDate = value;
         }
+      } else if (value === 'none' && this.oldTask.startDate) {
+        this.task.startDate = null;
+        this.$taskDrawerApi.updateTask(this.task.id, this.task).then(() => {
+          this.oldTask.startDate = this.task.startDate;
```
Comment (ahamdi) [inline task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue]: Is this needed ?

**Review by ahamdi — CHANGES_REQUESTED** [review]
Please update the PR title

File: task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue (hunk 1 above)
Comment (rdenarie) [reply task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue]: cc @hthabti Can you answer to Ali's question ? 
Thanks

**Comment by rdenarie** [comment]
@ahamdi as you made a comment on the PR, can you check if it is ok for you now ?
Thansk

**Review by ahamdi — APPROVED** [review]

**Pull request merged**